server/helpers/helper.js

```diff
@@ -1,13 +1,38 @@
 import bcrypt from 'bcryptjs';
+import moment from 'moment';
+
+/**
+ * @description formats a date using moment.js
+ * @function
+ * @param {string} date The date to format
+ * @returns {Date} Date
+ */
+export function formatDate(date) {
```

Comment:
any reason you're not complying with `ES6` standard here. You could easily `export const formatDate = (date) => {` here.

Just a thought.